packages/eslint-plugin/src/util/getWrappingFixer.ts

```diff
@@ -42,6 +42,13 @@ export function getWrappingFixer(
         code = `(${code})`;
       }
 
+      // check the inner node is used as return body of arrow function
+      if (isObjectExpressionInOneLineReturn(node, innerNode)) {
```

Comment:
q: should we do `else if` here instead? as we do not want code to be wrapped twice

Reply:
Yeah that's true. I fixed it!